### ISSUE TITLE
MixedOps-NIReconnect-1.5k-25m : HederaNodeValidator failed validation with error Possibly CATASTROPHIC failure in txn commit 

### ISSUE THREAD

**Issue by ljianghedera** [hashgraph/hedera-services#1431]
**Summary of the defect**
We've seeing the error messages list below in several tests listed. The increased TPS for reconnect test seems to exaggerate this issue.

There should be already a fix for this. Nevertheless, opening this issue to track it in case there are any ramifications.

**How to reproduce (if possible)**

A failure test report: https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1620652507164700

**Service logs (if applicable)**
```

---- HederaNodeValidator FAILED validation ----

Node 0 has exception:- Possibly CATASTROPHIC failure in txn commit :: sigMap {
Node 0 has exception:- java.lang.IllegalArgumentException: Track time 1620651725 for (165022,com.hedera.services.txns.schedule.ScheduleCreateTransitionLogic$$Lambda$1157/0x00007fbde99d28b0@1c5cb6fc) not later than 1620653459

...
```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

**Comment by Neeharika-Sompalli** [comment]
This issue is caused when Scheduled transactions expiration time is reduced from the initial time set. as mention in #1336, it will be resolved if a the queue is modified to `PriorityQueue` for expiring schedule transactions

**Comment by Neeharika-Sompalli** [comment]
This issue will be resolved by #1336
